src/settings/MappingProfiles/detailsSections/edit/OrderDetailSection/DonorInformation.js

```diff
@@ -0,0 +1,90 @@
+import React, {
+  useCallback,
+} from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+
+import {
+  Accordion,
+  RepeatableField,
+  Row,
+  Col,
+} from '@folio/stripes/components';
+
+import {
+  FieldDonor,
+  WithValidation,
+} from '../../../../../components';
+import { useFieldMappingRefValues } from '../../hooks';
+import { useDonorsQuery } from '../../../../../hooks';
+
+import { TRANSLATION_ID_PREFIX } from '../../constants';
+import {
+  getSubfieldName,
+  onAdd,
+  onRemove,
+} from '../../utils';
+import { DONORS_FIELD } from '../../../../../utils';
+
+export const DonorInformation = ({
+  initialFields,
+  setReferenceTables,
+}) => {
+  const DONORS_INDEX = 44;
```

Comment:
Why not move `DONORS_INDEX` and `DONOR_INFORMATION_MAP` outside the component?

Reply:
that's the common approach for all other sections

Reply:
I mean, you can left it in the same file, but move outside the component.
Each time the component is re-rendered, this requires unnecessary calculations:
```
const DONOR_INFORMATION_MAP = {
  DONOR_NAME: index => getSubfieldName(DONORS_INDEX, 0, index),
};
```